test/standalone/test-debuglet.js

```diff
@@ -17,9 +17,10 @@
 
 var assert = require('assert');
 var request = require('../auth-request.js');
-var logger = require('@google/cloud-diagnostics-common').logger;
-var config = require('../../src/config.js').debug;
+var loggerModule = require('@google/cloud-diagnostics-common').logger;
+var defaultConfig = require('../../src/config.js').debug;
 var Debuglet = require('../../src/agent/debuglet.js');
+var extend = require('extend');
```

Comment:
I have eventual ambitions of getting rid of the lodash dependency completely. The google-cloud-node prefers to use `is` and `extend` instead of `lodash`. This would lead to a smoother integration.